android/src/main/java/nl/fhict/happynews/android/PostManager.java

```diff
@@ -79,6 +79,49 @@ public void onCompleted(Exception e, Page result) {
             });
     }
 
+
+    /**
+     * Sends content of the first page to the FeedAdapter.
+     * @param context The Application context.
+     * @param listener Implementing the LoadListener Interface.
+     */
+    public void refresh(Context context, final LoadListener listener) {
```

Comment:
They are very similar indeed. Main difference being that ```refresh(Context context, final LoadListener listener)``` calls ```setPage()``` on the Adapter instead of ```addPage```. 

One solution can be to add a boolean switch parameter to the ```loadPage()```. But I think the function is already quite large so that might not be the cleanest solution...

Any other suggestions maybe?